package.json

```diff
@@ -1,87 +1,33 @@
 {
-  "name": "@moxy/eslint-config",
-  "description": "MOXY eslint configuration to be used across several JavaScript projects",
-  "version": "10.1.1",
-  "keywords": [
-    "eslint-config",
-    "eslintconfig",
-    "eslint",
-    "lint",
-    "linter",
-    "style",
-    "react-lint",
-    "react-eslint",
-    "vue-lint",
-    "vue-eslint"
-  ],
-  "author": "André Cruz <andre@moxy.studio>",
-  "homepage": "https://github.com/moxystudio/eslint-config",
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:moxystudio/eslint-config.git"
-  },
-  "license": "MIT",
-  "files": [
-    "es*.js",
-    "rules",
-    "addons"
-  ],
+  "name": "root",
+  "private": true,
   "scripts": {
```

Comment:
does it make sense to provide a run alias for lcov ? 

Something like: `"lcov": "open coverage/lcov-report/index.html"`

@satazor @dominguesgm

Reply:
Is this common in our projects? Honestly, I think it would be a good idea to provide that alias.

Reply:
Apparently it is, so I'll add it